aten/src/ATen/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ FILE(GLOB native_sparse_cpp "native/sparse/*.cpp")
 FILE(GLOB native_quantized_cpp
             "native/quantized/*.cpp"
             "native/quantized/cpu/*.cpp")
+FILE(GLOB native_quantized_h "native/quantized/*.h" "native/quantized/cpu/*.h")
```

Comment:
Are these still needed, given the `setup.py` change?

Reply:
Yes, or else the headers won’t be generated in the include/ dir.